measures/ApplyUpgrade/resources/constants.rb

```diff
@@ -2,7 +2,7 @@
 
 class Constants
   def self.NumApplyUpgradeOptions
-    return 25
+    return 30
```

Comment:
resstock seed project has 26 options now, so bump to 30 since I may add some more soon